libnavigation-ui/src/main/java/com/mapbox/navigation/ui/map/NavigationMapboxMap.java

```diff
@@ -434,21 +452,6 @@ public void addProgressChangeListener(@NonNull MapboxNavigation navigation) {
     }
   }
 
-  /**
-   * Can be used to automatically drive the map camera, puck and route updates
-   * when {@link TripSessionState#STARTED}.
-   * <p>
-   * Use {@link #removeProgressChangeListener()} when the session is finished to avoid leaks.
-   *
-   * @param navigation to add the progress listeners
-   * @param enableVanishingRouteLine determines if the route line should vanish behind the puck.
-   * @see MapboxNavigation#startTripSession()
-   */
-  public void addProgressChangeListener(@NonNull MapboxNavigation navigation, boolean enableVanishingRouteLine) {
```

Comment:
This is a semver-major change. We should deprecate the method instead.

Reply:
I can change this method signature back to the way it was.  There were a lot of conflicts when I rebased so it was difficult to remember what had previously been done to try and anticipate what was coming in this PR.

Reply:
I reverted the mistakes in my original rebase with master. I don't think I have any breaking semver changes any longer.

Reply:
We're going to keep this change and remove this in the 1.0 branch per our discussion.